tests/python/pytorch/sparse/test_sparse_matrix.py

```diff
@@ -504,6 +504,60 @@ def test_range_select(create_func, shape, dense_dim, select_dim, rang):
     assert torch.allclose(A_select_to_dense, dense_select)
 
 
+@pytest.mark.parametrize(
+    "create_func", [rand_diag, rand_csr, rand_csc, rand_coo]
+)
+@pytest.mark.parametrize("sample_dim", [0, 1])
+@pytest.mark.parametrize("index", [None, (0, 1, 3)])
+@pytest.mark.parametrize("replace", [0, 1])
+@pytest.mark.parametrize("bias", [0, 1])
+def test_sample(create_func, sample_dim, index, replace, bias):
+    ctx = F.ctx()
+    shape = (5, 5)
+    sample_num = 3
+    A = create_func(shape, 6, ctx)
+    A_row = A.row
+    A_col = A.col
+    A_val = torch.abs(A.val)
+    A = from_coo(A_row, A_col, A_val, shape)
```

Comment:
Why A is reconstructed?

Reply:
Because the values of matrix generated by function may contain negative elements which can not be used as bias array, I use 'abs' function to make them non-negative.